python/cudf/cudf/_libxx/stream_compaction.pxd

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from cudf._libxx.lib import *
+from cudf._libxx.lib cimport *
+
+cdef extern from "cudf/stream_compaction.hpp" namespace "cudf::experimental" nogil:
+    cdef enum duplicate_keep_option:
```

Comment:
Is this just a raw enum or an enum class in the cpp? If it's an enum class you need to do:
```
ctypedef enum duplicate_keep_option:
    ...
```

and then access as `duplicate_keep_option.KEEP_FIRST`